README.md

```diff
@@ -1,6 +1,6 @@
 # ENSNode
 
-ENSNode is a multichain indexer for ENS, powered by Ponder.
+ENSNode is a suite of applications and libraries powering ENS ecosystem.
```

Comment:
I think this should stay the same, or move to `ENSNode is a multichain indexer and suite of services for ENS`

Reply:
actually, given my PR proactively handles this, feel free to revert all changes to README files and docs files and we can do a separate once-over after docs changes land